app/src/Event/EventFormType.php

```diff
@@ -1,12 +1,14 @@
 <?php
 
-namespace Event;
+namespace JoindIn\Web\Event;
 
+use Event\Constraint;
```

Comment:
Should this be `use JoindIn\Web\Event\Constraint;`?

Reply:
actually it's redundant